docs/api-gateway/docs/06-01-api.md

```diff
@@ -35,8 +35,10 @@ This table lists all the possible parameters of a given resource together with t
 | Field   |      Mandatory      |  Description |
 |----------|:-------------:|------|
 | **metadata.name** |    **YES**   | Specifies the name of the exposed API |
-| **service.name**, **service.port** | **YES** | Specifies the name and the communication port of the exposed service. |
+| **spec.service.name**, **spec.service.port** | **YES** | Specifies the name and the communication port of the exposed service. |
 | **spec.hostname** | **YES** | Specifies the service's external inbound communication address. |
+| **spec.authenticationEnabled** | **NO** | If `false`, disables generation of the Istio Authentication Policy for the service entirely. Defaults to `true`. |
```

Comment:
If `true` and `spec.authentication` is set, authentication settings from `spec.authentication` will be used. 
If `true` and `spec.authentication` is not set or set to `[]`, authentication will be enabled and default authentication settings specified in api-controller environment variables will be used. 
If `false` authentication will be disabled and `spec.authentication` will be ignored.
If not set, authentication will depend only on `spec.authentication` field.

Reply:
IMO this kind of description is too long here. I was very brief on purpose here. Perhaps we should make a dedicated section to document that behaviour.

Reply:
It is quite long, but your brief description is IMHO not sufficient, as that flag is not only used to disable auth but, above all, it is used to enable default auth. 
Your description is also wrong, as it doesn't default to `true`. `not set` is not equal to `true` in that case.